10.0/contrib/wfcfg/standalone.xml

```diff
@@ -187,7 +187,7 @@
             </datasources>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
-            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
+            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="250" auto-deploy-zipped="${hot.deploy.enabled:false}" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```

Comment:
I realize, but if it takes 5 seconds to deploy a change, does it matter if the change was detected in 250ms or 1s?

you're talking about 6s vs 5.25s for the change to be visible.  so i'm wondering about the overhead of polling that frequently and whether it actually provides a benefit in terms of observed responsiveness.